test/e2e/funcs/feature.go

```diff
@@ -384,6 +384,17 @@ func DeleteResources(dir, pattern string) features.Func {
 	}
 }
 
+// CreateNamespace creates the given namespace
+func CreateNamespace(name string) features.Func {
+	return func(ctx context.Context, t *testing.T, config *envconf.Config) context.Context {
+		ns := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: name}}
+		if err := config.Client().Resources().Create(ctx, ns); err != nil {
+			t.Fatalf(fmt.Sprintf("error creating namespace %s", name), err)
+		}
+		return ctx
+	}
+}
+
```

Comment:
There is an almost identical function available in the e2e-framework, which we already use in a similar context [here](https://github.com/crossplane/crossplane/blob/444267e84783136daa93568b364a5f01228cacbe/test/e2e/install_test.go#L134).
If you feel there are reasons we should not use envfuncs.CreateNamespace, feel free to refactor also that usage and add here a comment with the reasons why this should be preferred over the other option.

Reply:
well, I have noticed that with each invocation of `envfuncs.CreateNamespace`, default namespace is set to that newly created namespace. Not sure if this is going to cause any issues in the feature, but I am gonna remove the added function.

Reply:
That's a valid reason indeed, I'd say to switch to a dedicated function